packages/@dcl/inspector/src/hooks/useAnalytics.ts

```diff
@@ -0,0 +1,54 @@
+import { Analytics } from '@segment/analytics-node'
+import { getConfig } from '../lib/logic/config'
+
+export enum Event {
+  ADD_ITEM = 'Add Item',
+  REMOVE_ITEM = 'Remove Item',
+  ADD_COMPONENT = 'Add Component',
+  REMOVE_COMPONENT = 'Remove Component'
+}
+
+export type Events = {
+  [Event.ADD_ITEM]: {
+    entityId: number
+    positionX: number
+    positionY: number
+    positionZ: number
+  }
+  [Event.REMOVE_ITEM]: {
+    name: string
+  }
+  [Event.ADD_COMPONENT]: {
+    type: string
+    parentEntityId: number
+  }
+  [Event.REMOVE_COMPONENT]: {
+    type: string
+    parentEntityId: number
```

Comment:
We should remove the entity ids from events since that's not really relevant for the data team. 

I would rename the `type` property in the ADD_COMPONENT and REMOVE_COMPONENT to be `componentName` to match the name of the property in the component.

The x,y,z information is not necessary for the ADD_ITEM event, but we need to provide some information about the item itself, like the item id and item name.

The REMOVE_ITEM should be REMOVE_ENTITY, and it does not need to have any payload. We are also missing the ADD_ENTITY and RENAME_ENTITY events.